lib/elixir/lib/kernel.ex

```diff
@@ -8,33 +8,26 @@ import :elixir_bootstrap
 
 defmodule Kernel do
   @moduledoc """
-  This module is the entry point of the Elixir programming language.
-
-  It provides the default functions and macros Elixir imports into your
-  environment. Those can be directly invoked in your code without the
-  module name:
-
+  `Kernel` is Elixir's set of core functionality, mainly consisting of:
```

Comment:
The first line of moduledoc should be a short self-contained summary. It will be displayed in some module summaries in documentation and similar.